conans/client/conan_api.py

```diff
@@ -626,7 +626,7 @@ def info_build_order(self, reference, settings=None, options=None, env=None,
         deps_graph, _ = self._graph_manager.load_graph(reference, None, graph_info, ["missing"],
                                                        check_updates, False, remotes,
                                                        recorder)
-        return deps_graph.build_order(build_order)
+        return deps_graph.build_order(build_order), deps_graph
```

Comment:
Just a note, this will break to all users using this via the conan_api. I know it is not stable, it is just a note, that some users will break. This is a very strong candidate method that might be used via the python conan_api by CI systems of users.

Reply:
> This is a very strong candidate method that might be used via the python conan_api by CI systems of users.

I didn't think about that ... so maybe it's not the best option.